src/routes/connect/Connect.svelte

```diff
@@ -27,6 +26,19 @@
     }
 
     bridge.sendOnReadyEvent();
+
+    const defaultPopupWidth = 360;
+    const defaultPopupHeight = 510;
+    const minimumPopupHeight = 210;
+    const minimumPopupWidth = 340;
+
+    if (window.outerHeight < minimumPopupHeight) {
+        self.resizeTo(self.outerWidth, defaultPopupHeight);
+    }
+
+    if (window.outerWidth < minimumPopupWidth) {
+        self.resizeTo(defaultPopupWidth, self.outerHeight);
+    }
```

Comment:
Please create a helper method that can be called from both routes and handles this logic in a single place